### ISSUE TITLE
Feature request: Is LDAP for a password backend possible?

### ISSUE THREAD

**Issue by saiab-systems** [gbif/ipt#1252]
Hi,

We're in the process of deploying an IPT at our organisation and are very impressed with it. What we like to avoid is staff having to use multiple usernames and passwords.

**Feature request**: Would it be possible to have LDAP (preferably) / RADIUS / Google authentication added to the IPT web application? 

We feel that this would be a security enhancement along with making a huge improvement from a user's perspective as they don't have another username and password to remember.

Best wishes,
Andrew

**Comment by kbraak** [comment]
Hi Andrew, 

Thank your for reporting this issue, we'll take it into consideration. 

A reminder that users can choose their preferred password by editing their account. See screenshot below. 

Best wishes,

Kyle

![screen shot 2016-01-12 at 14 58 46](https://cloud.githubusercontent.com/assets/5981100/12265416/307d73f6-b93d-11e5-9799-ab097351500d.png)

**Comment by waoki** [comment]
LDAP or CAS support (including assigning rights based on external information) would also be very valuable for us. The problem for us isn't that users have to remember (or set) another password, it's that we have to maintain yet another silo with separate user accounts.

**Comment by auspex** [comment]
Unless I misunderstand https://tomcat.apache.org/tomcat-7.0-doc/realm-howto.html, this isn't, and shouldn't be, a feature of the application (IPT, in this case). It appears to me that I should be configuring the Tomcat-level **Realm**, and the LDAP authentication will then become available to the app. It may require some modification of IPT to take advantage of it, though. I'll give it a try, anyway

**Comment by frafra** [comment]
I agree; it would be nice to have LDAP (and/or OAuth2) authentication :)
